ZukoAzula2016/src/org/team1540/zukoazula/AutonomousBase.java

```diff
@@ -112,6 +114,12 @@ protected void eject(float time, float driveBackSpeed) throws AutonomousModeOver
         Shooter.stopEvent();
     }
 
+    protected void ejectWhileDrivingBackDistance(float feet, float speed) throws AutonomousModeOverException, InterruptedException {
```

Comment:
Why not make this ejectWhileDrivingDistance(positive feet moves forwards, negative moves backwards) instead of having it do math to reverse? Also for consistency, the above "eject" function should be renamed as ejectWhileDrivingTime or something like that.